scheduler/domain/definitions.go

```diff
@@ -52,6 +52,10 @@ type JobDefinition struct {
 	Tasks     []TaskDefinition
 }
 
+func (jd *JobDefinition) String() string {
+	return fmt.Sprintf("jobType:%s, req:%s, tag:%s, basis: %s, tasks:%d", jd.JobType, jd.Requestor, jd.Tag, jd.Basis, len(jd.Tasks))
+}
+
```

Comment:
Somewhat of a tangent - what is happening with the Priority constructs defined in this file and used in various places?

Reply:
It's not used anymore, but I'm not taking it out due to the extent of the changes.  
I've created https://jira.twitter.biz/browse/MGCI-1310 for clean up issues.